src/Illuminate/Support/ViewErrorBag.php

```diff
@@ -33,7 +33,7 @@ public function hasBag($key = 'default')
      */
     public function getBag($key)
     {
-        return Arr::get($this->bags, $key, new MessageBag);
+        return Arr::get($this->bags, $key) ?: new MessageBag;
```

Comment:
See [this thread](https://github.com/laravel/framework/pull/10822#discussion-diff-44016926)